### PR TITLE
div. Features

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/controllers/org/gokb/UserController.groovy [hunk 1]
```diff
@@ -1,9 +1,13 @@
 package org.gokb
 
+import org.springframework.security.access.annotation.Secured
+
+@Secured(['IS_AUTHENTICATED_FULLY'])
 class UserController extends grails.plugin.springsecurity.ui.UserController {
 
   UserProfileService userProfileService
 
+  @Secured(['ROLE_ADMIN', 'IS_AUTHENTICATED_FULLY'])
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/controllers/org/gokb/UserController.groovy]: no need to secure method agains IS_AUTHENTICATED_FULLY if the whole controller is ...?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

**Comment by Moetez16** [comment]
PR updated

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**